From 9308498bb976bc139e043909f262cb5b4837c3fc Mon Sep 17 00:00:00 2001
From: Anna Born <anna.born@dfki.de>
Date: Mon, 2 Mar 2020 11:57:15 +0100
Subject: [PATCH] fix add pthread to cmake

---
 CMakeLists.txt | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 94ac2b7..f20e587 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -4,6 +4,8 @@ project(Util--)
 SET(PROJECT_VERSION 1.1)
 SET(API_VERSION 1)
 
+set( CMAKE_CXX_FLAGS " -pthread ")
+
 set (CMAKE_MODULE_PATH "${CMAKE_SOURCE_DIR}/cmake")
 FIND_PACKAGE(Boost COMPONENTS unit_test_framework)
 FIND_PACKAGE(Boost REQUIRED graph filesystem thread regex system)
-- 
2.17.1

